village/src/index.css

```diff
@@ -2,4 +2,9 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
```

Comment:
Great  job remembering the fundamentals here with  box sizing and resetting font size to use rem units